_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -30959,6 +30959,7 @@
 	},
 /obj/machinery/computer/pod/old/mass_driver_controller{
 	id = "toxinsdriver";
+	range = 6;
```

Comment:
This kind of change should be done through a variant type IMO.
No other var-edited mass drivers have this, and this is less maintainable.

Reply:
Hmm agreed

Reply:
Actually, let me have a look at all mass drivers and see if I can't group them together better.